### PR TITLE
Added Loudness filter options

### PR DESCRIPTION
**Pull request by JWahle**
Loudness filter can now be set through the GUI

### CODE REVIEW

File: src/common.js [hunk 1]
```diff
@@ -442,13 +443,23 @@ export class ParameterInput extends React.Component {
       tooltip: "Target frequency",
     },
     gain: { type: "float", desc: "gain", tooltip: "Gain in dB" },
+    high_boost: {
+      type: "float",
+      desc: "high_boost",
+      tooltip: "Volume boost for high frequencies when volume is at reference_level",
```
Comment (HEnquist) [inline src/common.js:449]: ...when volume is at reference_level - 20 dB

File: src/common.js [hunk 2]
```diff
@@ -442,13 +443,23 @@ export class ParameterInput extends React.Component {
...
     inverted: { type: "bool", desc: "inverted", tooltip: "Invert signal" },
     length: {
       type: "int",
       desc: "length",
       tooltip: "Number of coefficients to generate",
     },
+    low_boost: {
+      type: "float",
+      desc: "low_boost",
+      tooltip: "Volume boost for low frequencies when volume is at reference_level",
```
Comment (HEnquist) [inline src/common.js:461]: ...when volume is at reference_level - 20 dB

File: src/common.js [hunk 3]
```diff
@@ -478,6 +489,12 @@ export class ParameterInput extends React.Component {
       desc: "read_bytes_lines",
       tooltip: "Read up to this number of bytes or lines",
     },
+    reference_level: {
+      type: "float",
+      desc: "reference_level",
+      tooltip: "Volume level at which full low_boost/high_boost is applied." +
+        "<br>Boost is scaled down linearly to 0dB at reference_level + 20dB and above.",
```
Comment (HEnquist) [inline src/common.js:496]: "Volume level at which low_boost/high_boost is starting to be applied. Boost is scaled up linearly to reach the full value at reference_level - 20dB. Above reference_level only gain is applied."

File: src/filterlist.js [hunk 4]
```diff
@@ -101,6 +101,9 @@ class FilterParams extends React.Component {
     Volume: {
       Default: { ramp_time: 0.0 },
     },
+    Loudness: {
+      Default: { reference_level: 0.0, high_boost: 10, low_boost: 10, ramp_time: 0.0 },
```
Comment (HEnquist) [inline src/filterlist.js:105]: Could we put 0.2 as default for ramp_time, for both Loudness and Volume? Still fast enough not to be annoying, and avoids abrupt changes.

**Comment by HEnquist** [comment]
Great! I was expecting this one ;)

File: src/filterlist.js (hunk 4 above)
Comment (JWahle) [reply src/filterlist.js]: Sounds reasonable

**Comment by JWahle** [comment]
Haha - believe it or not, but until yesterday I have not thought about implementing this :)

**Comment by HEnquist** [comment]
Ah sorry why did I write 0.2... The unit for ramp time is ms, so it should be 200.0.
I also realized I firgot to add a validation check for the ramp time, probably should do that. Don't think anything good happens if someone puts a negative value!

**Comment by JWahle** [comment]
I changed it back to 200. More validation sounds good :)

**Pull request merged**